.prettierrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "printWidth": 80,
+  "tabWidth": 2,
+  "useTabs": false,
```

Comment:
Tips : I rather prefer using tabs, because each developers can set the shiftwidth of a tab in their own editor. IT will help accessibility for developers with a vision deficiency.

Reply:
I didn't look at this files, it's come from : https://github.com/actions/typescript-action/blob/main/.prettierrc.json#L4